db/fig_dev_seed_data/fig_dev_seed_data.rb

```diff
@@ -36,13 +36,13 @@ def yellow(text); colorize(text, 33); end
 
 #setup an admin user
 attrs = { password: password,
-          display_name: 'zooniverse_admin',
+          login: 'zooniverse_admin',
           email: 'no-reply@zooniverse.org' }
 admin = User.create(attrs) do |user|
   user.build_identity_group
```

Comment:
Can you please add the `user.admin = true` in here as well? Something that was forgotten about.